support-frontend/assets/pages/digital-subscription-landing/components/digitalSubscriptionLandingHeader.jsx

```diff
@@ -177,91 +178,114 @@ function getCopy(product: SubscriptionProduct, country: CountryGroupId) {
   };
 }
 
+const PaymentSelectionContainer = ({ dailyEditionsVariant }: { dailyEditionsVariant: boolean }) => (
+  <div className="payment-selection-container">
+    <LeftMarginSection>
+      <PaymentSelection dailyEditionsVariant={dailyEditionsVariant} />
+    </LeftMarginSection>
+  </div>
+);
+
+const anchorButton = () => (
+  <AnchorButton
+    id="qa-subscription-options"
+    aria-label="See Subscription options for Digital Pack"
+    onClick={sendTrackingEventsOnClick('options_cta_click', 'DigitalPack', null)}
+    icon={<SvgChevron />}
+    href="#subscribe"
+  >
+    See Subscription options
+  </AnchorButton>
+);
+
 function CampaignHeader(props: PropTypes) {
   const product: SubscriptionProduct = 'DigitalPack';
   const copy = getCopy(product, props.countryGroupId);
   return (
-    <ProductPagehero
-      appearance="campaign"
-      overheading={copy.heading}
-      heading={copy.subHeading}
-      modifierClasses={['digital-campaign']}
-      content={<AnchorButton id="qa-subscription-options" aria-label="See Subscription options for Digital Pack" onClick={sendTrackingEventsOnClick('options_cta_click', 'DigitalPack', null)} icon={<SvgChevron />} href="#subscribe">See Subscription options</AnchorButton>}
-      hasCampaign
-      dailyEditionsVariant={props.dailyEditionsVariant}
-    >
-      <div className="the-moment-hero">
-        <div className="the-moment-hero__copy">
-          <h2>A better way to fund journalism <br />
-            <span>A better way to read it
-            </span>
-          </h2>
-        </div>
+    <div>
```

Comment:
as a css novice, I just wonndered what this div is for? as normally they need a className for styling

Reply:
I'm not sure whether it's possible/desirable to break this down into more than one function (not necessarily in this PR anyway!)

Reply:
this is required to because react components need a containing html tag